<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This Widget is used to deal with general tasks realted to HBCI. 
All actions that can be completed here may also be solved in the guihbciwizard widget</comment>
 <class>GuiTools</class>
 <widget class="QWidget" name="GuiTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiToolsBase</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&lt;h3&gt;Toolbox for Experts&lt;/h3&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Setup Utilities</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox5_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>32767</width>
              <height>32767</height>
             </size>
            </property>
            <property name="title">
             <string>Preferences</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="textLabel1_3">
               <property name="text">
                <string>Time out for connection:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="lineEditTimeOut">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>32767</height>
                </size>
               </property>
               <property name="inputMask">
                <string>00; </string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="textLabel2_3">
               <property name="text">
                <string>sec</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="checkBoxShutdown">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>32767</height>
                </size>
               </property>
               <property name="text">
                <string>Automatic shutdown  
when MoneyPenny exits.</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonConsole">
            <property name="text">
             <string>&amp;Linux-Console</string>
            </property>
            <property name="shortcut">
             <string>Alt+L</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBuildPrinterDriverDatabase">
            <property name="text">
             <string>&amp;Build Printer Driver Database</string>
            </property>
            <property name="shortcut">
             <string>Alt+B</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Internet Connection Test</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>Test URL:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditUrl">
            <property name="text">
             <string>www.google.de</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textLabel2_2">
            <property name="text">
             <string>Test Port:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPort">
            <property name="text">
             <string>80</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="buttonConnectionTest">
            <property name="text">
             <string>&amp;Try to connect to URL:Port</string>
            </property>
            <property name="shortcut">
             <string>Alt+T</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxDataAnalysis">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Data Analysis</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonSQLQuery">
            <property name="text">
             <string>&amp;Send SQL-Query</string>
            </property>
            <property name="shortcut">
             <string>Alt+S</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCleanUp">
         <property name="title">
          <string>Clean Up</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="buttonDeleteLogFiles">
            <property name="text">
             <string>Delete OpenHBCI Log-Files</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Job Creation for Experts</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="buttonGetKeys">
            <property name="text">
             <string>Get &amp;Bank Public Key</string>
            </property>
            <property name="shortcut">
             <string>Alt+B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonKeyChange">
            <property name="text">
             <string>Change and Send Own Public &amp;Key</string>
            </property>
            <property name="shortcut">
             <string>Alt+K</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonGetSystemId">
            <property name="text">
             <string>Get S&amp;ystemId</string>
            </property>
            <property name="shortcut">
             <string>Alt+Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonGetAccounts">
            <property name="text">
             <string>Get &amp;Account List</string>
            </property>
            <property name="shortcut">
             <string>Alt+A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonIniLetter">
            <property name="text">
             <string>Print &amp;Ini-Letter</string>
            </property>
            <property name="shortcut">
             <string>Alt+I</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSaveOsConfig">
            <property name="text">
             <string>Save Operating System &amp;Configuration</string>
            </property>
            <property name="shortcut">
             <string>Alt+C</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="text">
        <string>&amp;Back</string>
       </property>
       <property name="shortcut">
        <string>Alt+B</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>lineEditTimeOut</tabstop>
  <tabstop>buttonConsole</tabstop>
  <tabstop>lineEditUrl</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>buttonConnectionTest</tabstop>
  <tabstop>buttonSQLQuery</tabstop>
  <tabstop>buttonGetKeys</tabstop>
  <tabstop>buttonKeyChange</tabstop>
  <tabstop>buttonGetSystemId</tabstop>
  <tabstop>buttonGetAccounts</tabstop>
  <tabstop>buttonIniLetter</tabstop>
  <tabstop>buttonSaveOsConfig</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
